src/Administration/Resources/app/administration/build/plugins.vite.ts

```diff
@@ -34,7 +34,7 @@ const isDev = VITE_MODE === 'development';
 
 // This env variable is provided by the symfony recipes
 const hasAdminRootEnv = !!process.env.ADMIN_ROOT;
-const host = process.env.VITE_HOST || (isInsideDockerContainer() ? getContainerIP() : undefined) || 'localhost';
+const host = process.env.HOST || process.env.VITE_HOST || (isInsideDockerContainer() ? getContainerIP() : undefined) || 'localhost';
```

Comment:
`VITE_HOST` is a more explicit env, therefore it should take presence:
```suggestion
const host = process.env.VITE_HOST || process.env.HOST || (isInsideDockerContainer() ? getContainerIP() : undefined) || 'localhost';
```
But shouldn't this more something that should be fixed at the dockware?

Reply:
Yeah I agree maybe this should be fixed at dockware rather!